examples/benchmarks/odspsnapshotfetch-perftestapp/webpack.config.cjs

```diff
@@ -46,17 +46,17 @@ module.exports = {
 	],
 	devServer: {
 		static: {
-			directory: path.join(__dirname, "/dist/fluid-loader.bundle.js"),
+			directory: path.join(__dirname, "/webpack/fluid-loader.bundle.js"),
```

Comment:
I think the static HTML emitted by 'expressApp.ts' expects the webpack bundle to be in the same directory as the node app?  (i.e., /lib/fluid-loader.bundle.js)

            <script type="text/javascript" src="/fluid-loader.bundle.js">

Reply:
The webpack is configured to be self-contained and independent from any tsc output. I don't recall seeing any webpack configs that use the local .js generated files. This is a strong motivation for me to have these paths completely separate.
The one thing I have seen is a jest config that uses the tsc .js results.
In many cases I think our tsc builds in app packages are just overhead. Having a tsc option could be nice for debugging, but don't represent a real scenario.